scripts/pi-hole/js/db_queries.js

```diff
@@ -310,13 +310,26 @@ $(document).ready(function() {
              "<'row'<'col-sm-4'l><'col-sm-8'p>>" +
              "<'row'<'col-sm-12'<'table-responsive'tr>>>" +
              "<'row'<'col-sm-5'i><'col-sm-7'p>>",
-        "ajax": {"url": APIstring, "error": handleAjaxError },
+        "ajax": {
+            "url": APIstring,
+            "error": handleAjaxError,
+            "dataSrc": function(data){
+                var new_data = new Array();
+                for(obj of data.data)
+                {
+                    obj[0] *= parseInt(1e6);
+                    obj[0] += dataIndex++;
+                    new_data.push(obj);
+                }
+                return new_data;
```

Comment:
This could be done more cleanly using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map).